cli/index.ts

```diff
@@ -216,6 +220,7 @@ const builtYargs = createYargsCli({
         }
       ],
       processFn: async argv => {
+        await maybeConfigureAnalytics();
```

Comment:
you haven't added this to all commands, why? why not just put it in `yargswrapper.ts`?

Reply:
I wanted to be careful here to not ask for terminal input on commands like `dataform compile --json`. So it's only on specific commands and dependent on some of those flags.

Reply:
OK ignore this, see other comment about detecting interactive terminals. I've moved this to all commands in the wrapper.